app/javascript/src/components/container/ContainerDatasetModalContent.js

```diff
@@ -625,16 +629,21 @@ export class ContainerDatasetModalContent extends Component {
             There are currently no attachments.
           </div>
         ) : (
-            <div className='mb-5'>
-            {attachmentGroups.Pending && attachmentGroups.Pending.length > 0
-              && renderGroup(attachmentGroups.Pending, 'Pending')}
-            {attachmentGroups.Original.length > 0 && renderGroup(attachmentGroups.Original, 'Original')}
-            {attachmentGroups.BagitZip.length > 0 && renderGroup(attachmentGroups.BagitZip, 'Bagit / Zip')}
-            {hasProcessedAttachments && Object.keys(attachmentGroups.Processed)
-              .map((groupName) => attachmentGroups.Processed[groupName].length > 0
-                && renderGroup(attachmentGroups.Processed[groupName], `Processed: ${groupName}`, groupName))}
-            {attachmentGroups.Combined.length > 0 && renderGroup(attachmentGroups.Combined, 'Combined')}
-          </div>
+          <>
+            <div className="mb-5">
+              {attachmentGroups.Pending && attachmentGroups.Pending.length > 0
+                && renderGroup(attachmentGroups.Pending, 'Pending')}
+              {attachmentGroups.Original.length > 0 && renderGroup(attachmentGroups.Original, 'Original')}
+              {attachmentGroups.BagitZip.length > 0 && renderGroup(attachmentGroups.BagitZip, 'Bagit / Zip')}
+              {hasProcessedAttachments && Object.keys(attachmentGroups.Processed)
+                .map((groupName) => attachmentGroups.Processed[groupName].length > 0
+                  && renderGroup(attachmentGroups.Processed[groupName], `Processed: ${groupName}`, groupName))}
+              {attachmentGroups.Combined.length > 0 && renderGroup(attachmentGroups.Combined, 'Combined')}
+            </div>
+            <Alert variant="warning" show={UserStore.isUserQuotaExceeded(filteredAttachments)}>
+              Uploading attachments will fail; User quota will be exceeded.
```

Comment:
Everywhere this message is shown, it would be good to inform the users how much space is available. Otherwise users have to find out with trial and error how big of a file they can upload?
`isUserQuotaExceeded` could give back the string including information regarding available space in case the quota is exceeded (e.g., `Uploading attachment(s) will fail; User quota (2 MB) will be exceeded.`).
In case the quota isn't exceeded `isUserQuotaExceeded` could give back an empty string.